lib/__tests__/standalone-ext.test.js

```diff
@@ -0,0 +1,125 @@
+'use strict';
+
+const path = require('path');
+const { existsSync } = require('fs');
+const standalone = require('../standalone');
+const safeChdir = require('../testUtils/safeChdir');
+const configColorNamed = require('./fixtures/config-color-named.json');
+
+const fixturesPath = path.join(__dirname, 'fixtures');
+
+describe('extensions', () => {
+	test('array', async () => {
+		const inputFiles = [
+			path.join(fixturesPath, 'no-empty-source.html'),
+			path.join(fixturesPath, 'extensions'),
+		];
+		const { results } = await standalone({
+			files: inputFiles,
+			customSyntax: 'postcss-scss',
+			extensions: ['css', 'scss'],
+			config: {
+				rules: { 'color-named': 'never' },
+			},
+		});
+
+		expect(results).toHaveLength(3);
+		expect(existsSync(path.join(fixturesPath, 'extensions', 'foo.json'))).toBe(true);
+		expect(results[0].source.endsWith('no-empty-source.html')).toBe(true);
+		expect(results[1].source.endsWith(path.join('extensions', 'bar.scss'))).toBe(true);
+		expect(results[2].source.endsWith(path.join('extensions', 'folder', 'qux.css'))).toBe(true);
```

Comment:
[suggest] I suggest the following refactoring to make the test code more readable.

First, define a tiny util:

```js
const fixture = (...items) => path.join(__dirname, 'fixtures', ...items);
```

Next, shorten codes with the util like this:

```suggestion
		expect(results[0].source).toBe(fixture('no-empty-source.html'));
		expect(results[1].source).toBe(fixture('extensions', 'bar.scss'));
		expect(results[2].source).toBe(fixture('extensions', 'folder', 'qux.css'));
```

By doing so, we can avoid repetition including `.endsWith` or `.toContain` and use `.toBe` for more accurate expectations.

Reply:
Iv seen something similar in another test. Should it be added to `testUtils/` instead?

Reply:
> Should it be added to testUtils/ instead?

Sounds nice.

Reply:
I don't think `replaceBackslashes` is needed.